<?xml version="1.0"?>

<launch>
	<node respawn="true" pkg="rovitis_phidgets" type="phidgets_stepper_bipolar" name="phidgets_stepper_bipolar" output="log">
		<param name="dev_serial_number" type="int" value="130505" />
		<param name="dev_serial_label" type="string" value="phidgets_stepper_bipolar" />
		<param name="frame" type="string" value="/phidget_sb" />
		<param name="wait_for_dev_delay" type="int" value="10000" />
		<param name="publish_frequency" type="int" value="5" />
		<param name="motor_acceleration" type="double" value="4400" />
		<param name="motor_velocity" type="double" value="800" />
		<param name="topic_name" type="string" value="/phidgets_stepper_bipolar_pose" />
		<param name="set_pose_service_name" type="string" value="phidgets_stepper_bipolar_set" />	
	</node>
</launch>
